### DISCUSSION TITLE
How to avoid the warning in Icepak.post.create_fieldplot_surface() ?

### DISCUSSION THREAD

**Discussion by Rungenwagen** [ansys/pyaedt#3580] [Q&A] [answered]
When I call the function Icepak.post.create_fieldplot_surface(), following warning always appears in my console. But the field plot is created in AEDT successfully. How can I avoid this warning?
```
C:\Python_envs\PyAEDT\lib\site-packages\pyaedt\generic\plot.py:33: UserWarning: The PyVista module is required to run some functionalities of PostProcess.
Install with

pip install pyvista

Requires CPython.
  "The PyVista module is required to run some functionalities of PostProcess.\n"
```

**Comment by Samuelopez-ansys** [comment, marked as answer]
Hi @Rungenwagen,

PyVista is not installed if you did just 'pip install pyaedt'

You can install all postprocessing pyaedt requirements doing:

pip install pyaedt[all]
